src/config.cc

```diff
@@ -33,6 +35,7 @@ void PrepareCommanLineParser(cmdline::parser& parser)
     parser.add<std::string>(config::Format,    'f', "output format of the AST (yaml|json)", false, "yaml", cmdline::oneof<std::string>("yaml", "json"));
     parser.add<std::string>(config::Type,      't', "type of the AST (refract|ast)", false, "refract", cmdline::oneof<std::string>("refract", "ast"));
     parser.add<std::string>(config::Sourcemap, 's', "export sourcemap AST into file", false);
+    parser.add(config::RefractSourceMap,       'r', "produce sourcemap for refract AST");
```

Comment:
Couldn't we merge `r` and `s` option? I don't like two options for basically the same thing.